src/app.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const cors = require('cors');
+const { authRouter } = require('./routes/auth.route.js');
+const { userRouter } = require('./routes/user.route.js');
+const { errorMiddleware } = require('./middlewares/errorMiddleware.js');
+const cookieParser = require('cookie-parser');
+
+const app = express();
+
+app.use(
+  cors({
+    origin: process.env.CLIENT_HOST,
```

Comment:
Ensure that `process.env.CLIENT_HOST` is correctly set in your environment to avoid CORS issues. This is important for allowing requests from the specified origin.